### DISCUSSION TITLE
PWA 도입에 대해 어떻게 생각하는가!

### DISCUSSION THREAD

**Discussion by hozzijeong** [woowacourse-teams/2023-pium#395] [개발 논의]
5차 데모데이가 끝나고 사용자에게 제공할 수 있는 최고한의 기능은 다 만들었다고 생각했습니다.

기술적으로 무엇인가를 더 개발하기 위해서는 사용자의 실제 피드백이 필요한 상황입니다. 하지만 사용자를 모집하기가 쉽지 않습니다.

따라서 기술적으로 좀 더 시도해 볼 수 있는게 뭐가 더 있을까 생각을 했을 때 PWA를 통한 스토어 배포입니다.

스토어 배포를 하게 된다면 디바이스의 기술인 "알림"을 사용할 수 있고, 리마인더 알림을 줄 수 있습니다.

![image](https://github.com/woowacourse-teams/2023-pium/assets/50974359/1d0035f9-6265-4319-8cf7-5eadd0faf9fe)

현재 서비스에 PWA 설정을 하기 위해선 위에 요구사항을 만족해야 합니다.

https://web.dev/pwa-checklist/?utm_source=lighthouse&utm_medium=devtools

**Comment by WaiNaat** [comment]
매력적이긴 한데 남은 시간동안 가능한 기능인지는 모르겠어요
PWA의 컨셉 중 하나인 오프라인 동작을 위한 작업을 제대로 이해하고 사용할 수 있을까요?

**Reply by hozzijeong** [reply]
모든 기능을 완벽하게 구현한다기보다 어느 정도 기준을 두고 개발을 해야 할 것 같아요...! 

PWA를 통해서 개발을 하는 이유는 푸시 알림과 자동적인 설치를 통해서 접근성을 높이기 위함이었습니다. 

지금 생각해 봤을 때 오프라인 동작을 위한 작업은... 약간 우선순위가 뒤로 밀릴수 있지 않을까 생각이 듭니다 (물론 학습 해봐야 알 것 같아요)

**Comment by Choi-JJunho** [comment]
개발 완료로 close하겠습니다.

- https://github.com/woowacourse-teams/2023-pium/pull/414
- https://github.com/woowacourse-teams/2023-pium/pull/411
